cfme/tests/intelligence/reports/test_tenant_quota_reports.py

```diff
@@ -53,14 +67,48 @@ def test_queue_tenant_quota_reports(appliance, request, set_and_get_tenant_quota
         casecomponent: Reporting
         caseimportance: high
         initialEstimate: 1/10h
+        tags: report
+        setup:
+            1. Set tenant quota
+            2. Create the report
+        testSteps:
+            1. Compare the 'total quota' data in the reports and the quota that was set initially.
+        expectedResults:
+            1. Both the data must be same.
     """
-    tenant_report = appliance.collections.reports.instantiate(
-        type="Tenants", subtype="Tenant Quotas", menu_name="Tenant Quotas"
-    ).queue(wait_for_finish=True)
-    request.addfinalizer(tenant_report.delete)
-
     report_data = dict()
     for row in tenant_report.data.rows:
         if row["Quota Name"] in property_mapping.values():
             report_data[row["Quota Name"]] = row["Total Quota"]
     assert report_data == set_and_get_tenant_quota
+
+
+@test_requirements.report
+@pytest.mark.tier(3)
+def test_report_fullscreen_enabled(request, tenant_report, set_and_get_tenant_quota):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: low
+        initialEstimate: 1/12h
+        tags: report
+        setup:
+            1. Navigate to Cloud > Intel > Reports > All Reports
+        testSteps:
+            1. Select a report that would generate a populated report on queueing.
+            Queue it, navigate to it's `Details` page, click on Configuration and
+            check the `Show Fullscreen Report` option.
+            2. Select a report that would generate an empty report on queueing.
+            Queue it, navigate to it's `Details` page, click on Configuration and
+            check the `Show Fullscreen Report` option.
+        expectedResults:
+            1. `Show Fullscreen Report` option is Enabled.
+            2. `Show Fullscreen Report` option is Disabled.
+    """
+    view = navigate_to(tenant_report, "Details")
+    assert not view.configuration.item_enabled("Show full screen Report")
+    non_empty_report = tenant_report.parent.parent.queue(wait_for_finish=True)
+    request.addfinalizer(non_empty_report.delete)
+    view = navigate_to(non_empty_report, "Details")
+    assert view.configuration.item_enabled("Show full screen Report")
```

Comment:
@valaparthvi 
I can see two assert statements here.
Can we go with ```soft_assert```?

Reply:
Good catch! Will test that out!